lib/core/index.tsx

```diff
@@ -42,3 +48,19 @@ export const Main = forwardRef(
   )
 );
 Main.displayName = "Main";
+
+/* Side
+ * An element that houses the side menu and handles resizing on smaller screens.
+ */
+export const Side = ({ children, style, open, minimised }: SideProps) => (
+  <div
+    className={`side-interface${valOr(open === 1, "zero", "minus")}${valOr(
```

Comment:
```suggestion
    className={`side-interface${valEmpty(!open, "hidden")}${valOr(
```
Can `open` be changed to a boolean, and only include `hidden` class if it is not open.